iOS Client/Models/MBDeserializable.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  MBDeserializable.swift
+//  iOS Client
+//
+//  Created by Alex Ramey on 10/5/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import CoreData
+
+protocol MBDeserializable {
+    static func deserialize(json: NSDictionary, intoContext managedContext: NSManagedObjectContext) -> Error?
+}
+
+//protocol ArticleLoadState {
```

Comment:
need to delete this comment (and maybe this entire file, I don't think we need it . . .)